pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -0,0 +1,297 @@
+        "psql": "4.2.7",
+        "comments": "plan error:Unsupported SQL aggregation kind: {}. Only splittable aggregation functions are supported! [SINGLE_VALUE]",
+        "ignored": true,
+        "description": "outer aggregate",
+        "sql": "select  (select max((select i.int_col from {tbl} i where i.int_col = o.int_col))) from {tbl} o;"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "description": "nested",
+        "comments": "this should throw an error",
+        "sql": "SELECT min(max(int_col)) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "description": "nested",
+        "comments": "this should throw an error",
+        "sql": "SELECT (SELECT max(min({int_col})) FROM {tbl}) from {tbl};"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "parsing error:ORDER BY unexpected",
+        "description": "filter",
+        "sql": "SELECT min(double_col ORDER BY int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "NumberFormatException: For input string: \"a\"",
+        "description": "filter",
+        "sql": "SELECT min(string_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "class java.lang.Double cannot be cast to class java.lang.Boolean",
+        "description": "filter",
+        "sql": "SELECT min(bool_col) FROM {tbl}"
+      }
+    ]
+  },
+  "aggregate_filter": {
+    "tables": {
+      "tbl": {
+        "schema": [
+          {
+            "name": "int_col",
+            "type": "INT"
+          },
+          {
+            "name": "double_col",
+            "type": "DOUBLE"
+          },
+          {
+            "name": "string_col",
+            "type": "STRING"
+          },
+          {
+            "name": "bool_col",
+            "type": "BOOLEAN"
+          }
+        ],
+        "inputs": [
+          [
+            2,
+            300,
+            "a",
+            false
+          ],
+          [
+            2,
+            400,
+            "a",
+            true
+          ],
+          [
+            3,
+            100,
+            "b",
+            true
+          ],
+          [
+            0.001,
+            1,
+            "b",
+            false
+          ],
+          [
+            101,
+            1.01,
+            "c",
+            false
+          ],
+          [
+            150,
+            1.5,
+            "c",
+            true
+          ],
+          [
+            175,
+            1.75,
+            "c",
+            true
+          ],
+          [
+            -10000,
+            1.75,
+            "c",
+            false
+          ],
+          [
+            -2,
+            0.5,
+            "c",
+            false
+          ]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "psql": "4.2.7",
+        "description": "filter",
```

Comment:
can we update these descriptions to be more meaningful? it's not clear to me what each of these are testing

Reply:
Updated the description.